### PR TITLE
fromJson was replaced by JSON.parse with custom reviver fn

### PR DESCRIPTION
**Pull request by idooo**
Issue #203 
'true' and 'false' object values will be converted to booleans during .get() that will increase consistency between get/set functions.

I assume this is breaking change, so if this PR would be merged - we should notify developers about that

### CODE REVIEW

File: dist/angular-local-storage.js [hunk 1]
```diff
@@ -136,7 +135,13 @@ angularLocalStorage.provider('localStorageService', function() {
       }
     }());
 
-
+    // Reviver function for JSON.parse that will be called
+    // for every key and value at every level of the final string -> JSON transformation
+    var reviver = function(key, value) {
+      if (value === 'true' || value === 'false') { return (value === 'true'); }
```
Comment (a8m) [inline dist/angular-local-storage.js]: - Please use function declaration instead.
- There's no point to wrap it with `{}`.
- Also, these parentheses unnecessary here.

**Comment by a8m** [comment]
Looks good! 
I just added some minor style changes.
Thanks.

**Comment by idooo** [comment]
Done. Thanks

**Comment by a8m** [comment]
That's great!
Thanks @idooo

**Pull request merged**